modules/ROOT/pages/release-notes.adoc

```diff
@@ -41,7 +41,9 @@ This will provide the flexibility for "import heavy" use case users to be able t
 
 *Enhancements*
 
-{jira-url}/CBG-584[*CBG-584*] SyncGateway now supports scenarios where alternate addresses are provided on Couchbase Server.
+* Sync Gateway now supports CentOS8 and RELH8
```

Comment:
This was already merged, but there's a typo on `RELH8`. Should be `RHEL8`.

Reply:
DOH! I'm on it.